finetune/encoding.py

```diff
@@ -118,111 +94,23 @@ def __setitem__(self, key, value):
         self.encoder[key] = value
 
     def bpe(self, token):
```

Comment:
I think this is not necessary to be in the base class as I think it only ever called from _encode, I think it can be given a more generic name if you want to keep it.